modules/nav-menu.js

```diff
@@ -0,0 +1,39 @@
+// Navigation menu
+/* eslint-disable import/prefer-default-export */
```

Comment:
- [ ] As required by the previous reviewer it's better to fix linters and not disable it. Please do not disable your linters, fixing linters helps to write a clear and good codebase 🙃 🙃